docs/cookbook/shop/disabling-localised-urls.rst

```diff
@@ -63,3 +63,26 @@ With:
         product_grid:
             include_all_descendants: true
         locale_switcher: storage
+
+**3.** Adjust Sylius' Security Shop Regex in the ``config/packages/security.yaml``
+
+Add on the top of the file:
+
+.. code-block:: yaml
+
+    # config/packages/security.yaml
+
+    parameters:
+        sylius.security.shop_regex: "^(?:/(?!%sylius_admin.path_name%|api/.*|api$|media/.*)[^/]++)?"
+
+So the file should look like this:
+
+.. code-block:: yaml
+
+        # config/packages/security.yaml
+
+        parameters:
+            sylius.security.shop_regex: "^(?:/(?!%sylius_admin.path_name%|api/.*|api$|media/.*)[^/]++)?"
+
+        security:
+        # rest of the file
```

Comment:
```suggestion
    # config/packages/security.yaml

    parameters:
        sylius.security.shop_regex: "^(?:/(?!%sylius_admin.path_name%|api/.*|api$|media/.*)[^/]++)?"

    security:
    # rest of the file
```
I feel like this part is redundant, but adding it won't do any harm so lets keep it.